libraries/rush-lib/src/logic/versionMismatch/VersionMismatchFinder.ts

```diff
@@ -134,7 +134,7 @@ export class VersionMismatchFinder {
       truncateLongPackageNameLists?: boolean | undefined;
     }
   ): void {
-    if (rushConfiguration.ensureConsistentVersions || options.isRushCheckCommand) {
+    if (options.subspace?.shouldEnsureConsistentVersions || options.isRushCheckCommand) {
```

Comment:
When `options.subspace` is omitted, then `?.` prevents `shouldEnsureConsistentVersions` from being considered at all.